x/globalfee/types/query.pb.go

```diff
@@ -6,7 +6,7 @@ package types
 import (
 	context "context"
 	fmt "fmt"
-	_ "github.com/gogo/protobuf/gogoproto"
+	_ "github.com/cosmos/gogoproto/gogoproto"
```

Comment:
Chiming in here. If this is for v0.45, the gogo-proto version is buf.yaml should be pinned to `bee5511075b7499da6178d9e4aaa628b`. (https://buf.build/cosmos/gogo-proto/file/bee5511075b7499da6178d9e4aaa628b:gogoproto/gogo.proto#L36). Latest uses our fork (https://buf.build/cosmos/gogo-proto/file/5e5b9fdd01804356895f8f79a6f1ddc1:gogoproto/gogo.proto#L36) and is only needed from v0.47+.

Reply:
thank you very much!